src/search/pruning/stubborn_sets.h

```diff
@@ -9,14 +9,6 @@ inline FactPair find_unsatisfied_condition(
     const std::vector<FactPair> &conditions, const State &state);
```

Comment:
I think the function `find_unsatisfied_condition` could be moved to the action-centric class. The atom-centric only uses the `find_unsatisfied_precondition` function. (Note the subtle difference.)

Reply:
The implementation of `find_unsatisfied_precondition` uses `find_unsatisfied_condition`.

Reply:
Right. My IDE didn't show me this.